### DISCUSSION TITLE
关于新版的功能建议

### DISCUSSION THREAD

**Discussion by WangNingkai** [WangNingkai/OLAINDEX#348] [Ideas]
### 功能建议

- 后台重建缓存
- 文件加密、隐藏
- 直链优化
- 添加 swoole 支持
- 图片等媒体翻页预览
- etc.

**Comment by shaopak** [comment]
6.0 希望可以加入路径兼容模式；目前只单账户使用，路径兼容模式会简化网址，也不会因为以后更换账户而让网址失效

**Reply by WangNingkai** [reply]
访问路径的话已经可以使用兼容路径，但具体的访问，受多账号影响目前还不支持

**Comment by shaopak** [comment]
目前文件流里面填入格式后，大于50M则无法显示预览界面，比如zip格式，造成无法提取短链；
目前解决办法就是在图片一栏填入相应格式。

建议解除这个大小限制，在预览界面对于不支持显示内容的文件显示格式图标，比如压缩文件的压缩图标

**Comment by misaka123456** [comment]
希望图片在文件夹目录添加预览功能，像4.0一样点击预览按钮即可在当前目录下打开或者说在当前页前弹出一个悬浮页的效果（跳转到新页面打开文件也应该保留），并且可以通过上一张下一张切换，视频和音乐也希望有类似的功能。

**Comment by xiaorui6668** [comment]
![~AQ6FT MZH I 98$(50`F96](https://user-images.githubusercontent.com/76908807/103480655-6d972f80-4e10-11eb-9733-d7f06506b0db.png)
新版本建议加入加密选项功能，很需要这个功能

**Comment by misaka123456** [comment]
想问一下未来有支持阿里的teambition网盘的计划吗？

**Reply by WangNingkai** [reply]
这里看一下另外一个项目 https://github.com/WangNingkai/TeambitionIndex

**Comment by feiwujia** [comment]
希望加密文案可以支持Markdown，4.0版本支持，后面就不支持了。

**Comment by honorcnboy** [comment]
绑定多个网盘时，前端选择网盘选项里是按添加顺序显示的，是否可以调整显示的顺序？

**Comment by honorcnboy** [comment]
默认主题，文件名无法完全显示 [6.0版本]：
在默认主题下：
PC端：文件名过长，无法完全显示
手机端：仅可显示文件名前6个汉字或10多个字符

是否可以通过修改代码，实现名称自动换行，以满足显示。

请老王指点一下 @WangNingkai

**Comment by Dage1819** [comment]
1、建议新版本下载按钮进行一下加密，&download=1这种链接容易导致别人恶意盗取永久直连分享。建议把1改成变量值，可以每天或者每小时变化一次。
2、mp4视频分享会自动播放，建议关掉，如果想下载的话会因为浏览器缓存视频下载速度很慢。
3、短链接分享，建议访问的时候别跳转到真实链接。
4、加密方式建议简化为账号修改备注那样直接在对应目录填写
5、建议添加修改文件命名和移动文件功能。
6、反代功能，cf wokers。可以在od线路抽风的时候选用备用路径下载（即反代）
以上更新建议，希望老王能完善一下。定扫码支持。

@WangNingkai

**Comment by ysn2382617094wsw** [comment]
能不能出个视频教程教一下全程部署啊，小白表示不会部署太难受了
